src/components/backend-ai-scaling-group-list.ts

```diff
@@ -576,6 +839,25 @@ export default class BackendAIScalingGroupList extends BackendAIPage {
             </mwc-button>
        </div>
       </backend-ai-dialog>
+      <backend-ai-dialog id="keypair-detail-dialog" fixed backdrop blockscrolling>
```

Comment:
Is this element id is supposed to have "keypair-detail-dialog"?

Reply:
And most of detail dialog contains not only for information that didn't covered in table UI, but it also contains other essential information. Please checkout user detail dialog or vfolder detail dialog. You may get a clue from there.